### DISCUSSION TITLE
I can not access 'tools' anymore

### DISCUSSION THREAD

**Discussion by ON5MF** [Nachtzuster/BirdNET-Pi#249] [Q&A] [answered]
hi,
I always accessed the tools tab with the default username 'birdnet' and without a password.
For some reason I can't get in anymore.
Any clues?

update: my "/etc/birdnet/birdnet.conf" does not contain a password

**Comment by stevedee** [comment, marked as answer]
Take a look at: https://github.com/Nachtzuster/BirdNET-Pi/issues/243
